### PR TITLE
Port Numbers Cheat Sheet: Update port numbers

### PR DESCRIPTION
**Pull request by vikashvverma**
Updated list of port numbers in cheat sheet.

IA Page: [https://duck.co/ia/view/default_port_numbers_cheat_sheet](https://duck.co/ia/view/default_port_numbers_cheat_sheet)

### CODE REVIEW

**Comment by daxtheduck** [comment]
## [Default Port Numbers Cheat Sheet](https://duck.co/ia/view/default_port_numbers_cheat_sheet)

**Description**: This instant answer shows the default port numbers that are used in computer networks

**Example Query**: [Default Port Numbers Cheat Sheet](https://beta.duckduckgo.com/?q=Default Port Numbers Cheat Sheet), [](https://beta.duckduckgo.com/?q=)

**Tab Name**: Answer

**Source**: https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/default_port_numbers_cheat_sheet)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/default_port_numbers_cheat_sheet)*

File: share/goodie/cheat_sheets/json/default-port-numbers.json [hunk 1]
```diff
@@ -59,8 +59,23 @@
...
+        }, {
+            "val": "NFS Mount (NFS)",
+            "key": "1048"
+        }, {
+            "val": "IBM AIX Network Installation Manager (NIM)",
+            "key": "1058"
+        }, {
             "val": "SOCKS Proxy",
-            "key": "1080"
+            "key": "1180"
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/default-port-numbers.json]: ? I believe this should be `1080`: http://www.speedguide.net/port.php?port=1080

Comment (vikashvverma) [inline share/goodie/cheat_sheets/json/default-port-numbers.json]: Yeah, you're right.

fixed that.

**Comment by mintsoft** [comment]
Thanks @vikashvverma

**Pull request merged**